docs/release-notes/known-issues.md

```diff
@@ -8,6 +8,20 @@ For Elastic {{observability}} known issues, refer to [Elastic Observability know
 
 For Elastic Security known issues, refer to [Elastic Security known issues](docs-content://release-notes/elastic-security/known-issues.md).
 
+::::{dropdown} The connection between agentless integrations and {{fleet-server}} is broken
+
+Applies to: {{stack}} 9.0.0-9.1.5
```

Comment:
can this be marked as `< 9.1.6` rather than this range?

(agentless became available sometime in late 8.x, not just from 9.0)